e2e-tests/.eslintrc.js

```diff
@@ -7,13 +7,18 @@ module.exports = {
   },
   plugins: ['cypress'],
   rules: {
-    'max-len': ['error', 160, 2, {
-      ignoreUrls: true,
-      ignoreComments: false,
-      ignoreRegExpLiterals: true,
-      ignoreStrings: true,
-      ignoreTemplateLiterals: true,
-    }],
+    'max-len': [
```

Comment:
The issue also resides with conflicting prettier and Eslint rules.
If you can follow DTFS guide on this (big task) and then you can decomment `prettier:all` in `lint.yml`.